docs/README.md

```diff
@@ -0,0 +1,161 @@
+## Run a full diracx demo
+
+This will allow you to run a demo setup.
+
+The code changes will be reflected in the demo.
+
+Requirement: docker, internet
+
+```bash
+# Clone the diracx repository
+git clone git@github.com:<yourfork>/diracx.git
+
+# Clone the diracx-chart repository
+git clone git@github.com:<yourfork>/diracx-charts.git
+
+# Run the
+diracx-charts/run_demo.sh diracx/
+```
+
+## Create the dev environment
+
+This will help you setup a dev environment to run the unit tests
+
+Requirements: conda, git
+
+```bash
+# Clone the diracx repository
+git clone git@github.com:DIRACGrid/diracx.git
+cd diracx
+
+# Create the mamba environment
+mamba env create --file environment.yml
+conda activate diracx-dev
+
+# Make an editable installation of diracx
+
+pip install -e .
+
+# Install the patched DIRAC version
+pip install git+https://github.com/chaen/DIRAC.git@chris-hack-a-ton
+
+# Enable pre-commit
+mamba install pre-commit
+pre-commit install
+```
+
+### Run the test
+
+Run the unit tests
+
+```bash
+# In the `diracx` folder
+pytest
+mypy .
+pre-commit run --all-files
+```
+
+### Run a local instance of diracx
+
+This only runs the diracx server, not any dependency like external IdP.
+
+
+```bash
+# In the `diracx` folder
+./run_local.sh
```

Comment:
In my case I should correct following in the script: 
tmp_dir="/Users/ruslan/BelleII/DIRACX/dev/tmp" #explicitly name tmp dir
export DIRACX_CONFIG_BACKEND_URL="git+file://${tmp_dir}/cs_store/initialRepo" #add this env var